developer/src/kmc-kmn/build.sh

```diff
@@ -65,6 +65,17 @@ function do_build() {
 
 function do_test() {
   copy_deps
+
+  # We want to compare the key cap values from both KMW and Developer and ensure
+  # that all three are in sync. We'll copy the relevant source files and patch
+  # them in directly. Builder's constants.js is not an ES6 module, so we hackily
+  # patch that here.
+  echo 'export const builder = {specialCharacters:{}}' > ./test/kmw/_imported_constants.js
+  # shellcheck disable=SC2016
+  echo 'function $(v) {v()}' >> ./test/kmw/_imported_constants.js
+  cat "${KEYMAN_ROOT}/developer/src/tike/xml/layoutbuilder/constants.js" >> ./test/kmw/_imported_constants.js
+  cp "${KEYMAN_ROOT}/web/src/engine/osk/src/specialCharacters.ts" ./test/kmw/_imported_specialCharacters.ts
```

Comment:
I'm wondering if it would be easier maintainable if the filename would contain the source?

```suggestion
  cat "${KEYMAN_ROOT}/developer/src/tike/xml/layoutbuilder/constants.js" >> ./test/kmw/_imported_layoutbuilder_constants.js
  cp "${KEYMAN_ROOT}/web/src/engine/osk/src/specialCharacters.ts" ./test/kmw/_imported_osk_specialCharacters.ts
```